gemini/sample-apps/image-bash-jam/tts.sh

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+##############################################################################
+# word to text via CURL :)
+# Docs https://cloud.google.com/text-to-speech/docs/create-audio-text-command-line
+# Before you run, pls execute:
+#
+# 	gcloud auth application-default login
+# 	gcloud auth application-default set-quota-project "$PROJECT_ID"
+#
+##############################################################################
+
+. .envrc
+
+set -euo pipefail
+
+#PROJECT_ID=$(gcloud config get-value project)
```

Comment:
Can you remove all of the commented out lines?

Reply:
Great point. Doing so for most line.

I leave this since i learnt it the hard way and its super interesting:
#DEFAULT_GENDER='MALE' doesnt work in italian -> dflt is it-IT-Neural2-A

I've removed all the garbage and moved above the sample JSON which seems great documentation to me. PTAL